src/main/java/com/actiontech/dble/services/mysqlauthenticate/MySQLBackAuthService.java

```diff
@@ -130,7 +130,7 @@ protected void handleInnerData(byte[] data) {
                     break;
             }
         } catch (Exception e) {
-            LOGGER.warn(e.getMessage(), e);
```

Comment:
？

Reply:
after modifying the constructor.
e.getMessage() return the msg
e.toString() return the msg+code